### PR TITLE
Add config section for each coin and include minloansize for each coin

### PR DESCRIPTION
**Pull request by laxdog**
Add config section for each specific coin (alternate layout to coinconfig) and include minloansize in the config for each coin

## Description
Broke out the config for the coins to allow a separate ConfigParser section for each coin. If coinconfig is not found in the default.cfg, then it will look for these individual sections instead.

The minloansize has been added to these sections, so can be configure per coin. This does not change the coinconfig logic and means we are still backwards compatible with older config files. 

It should also make it the config a bit cleaner for people who have multiple alts.

## TESTING STAGE
Tested with previous pull request and no logic changes created in between. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] **I have read CONTRIBUTING.md**
- [x] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [x] **My code adheres to the [code style of this project.](https://poloniexlendingbot.readthedocs.io/en/latest/contributing.html)**
- [x] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [x] **I have updated the config file accordingly if my change requires a new configuration setting or changes an existing one.**
- [x] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -234,6 +238,22 @@ Config per Coin
 - MAXTOLEND, MAXPERCENTTOLEND, and MAXTOLENDRATE refer to their respective settings above, but are unique to the specified coin specifically.
 - There can be as many different coins as you want in coincfg, but each coin may only appear once.
 
+**Separate coin sections**
+
+This is an alternative layout for the coin config mentioned above. It provides the ability to change the minloansize per coin, but is otherwise identical in functionality.
+To use this configuration, make sure to comment out the line where coincfg is defined, then add a secion for each coin you wish to configure. These sections should come
```
Comment (Evanito) [inline docs/configuration.rst]: typo "secion" > "section"

File: docs/configuration.rst [hunk 2]
```diff
@@ -234,6 +238,22 @@ Config per Coin
...
+
+This is an alternative layout for the coin config mentioned above. It provides the ability to change the minloansize per coin, but is otherwise identical in functionality.
+To use this configuration, make sure to comment out the line where coincfg is defined, then add a secion for each coin you wish to configure. These sections should come 
+at the end of the file, after the other options for the bot.
+Configuration should look like this::
+
+    [BTC]
+    minloansize = 0.01
+    mindailyrate = 0.1
+    maxactiveamount =
```
Comment (Evanito) [inline docs/configuration.rst]: maxactiveamount = 1?

File: modules/Configuration.py [hunk 3]
```diff
@@ -73,9 +74,41 @@ def get_coin_cfg():
...
+                    raise
     return coin_cfg
 
 
+def get_min_loan_sizes():
+    min_loan_sizes = {}
+    for cur in FULL_LIST:
+        if config.has_section(cur):
+            try:
+                min_loan_sizes[cur] = config.get(cur, 'minloansize')
```
Comment (Evanito) [inline modules/Configuration.py]: Must cast as Decimal, and I recommend using the Configuration module's built-in get() command to enforce maxes and mins.

File: docs/configuration.rst [hunk 4]
```diff
@@ -234,6 +238,22 @@ Config per Coin
 - MAXTOLEND, MAXPERCENTTOLEND, and MAXTOLENDRATE refer to their respective settings above, but are unique to the specified coin specifically.
 - There can be as many different coins as you want in coincfg, but each coin may only appear once.
 
+**Separate coin sections**
+
+This is an alternative layout for the coin config mentioned above. It provides the ability to change the minloansize per coin, but is otherwise identical in functionality.
+To use this configuration, make sure to comment out the line where coincfg is defined, then add a secion for each coin you wish to configure. These sections should come 
+at the end of the file, after the other options for the bot.
```
Comment (Evanito) [inline docs/configuration.rst]: Put this notice in bold so that people notice it at a glance, as it is very important.

Comment (Evanito) [inline docs/configuration.rst]: or in:
```
.. warning:: These sections must come at the end of the file, after the other options for the bot.
```

File: modules/Configuration.py (hunk 3 above)
Comment (laxdog) [inline modules/Configuration.py]: Fixed all the doc updates, I don't get the last fix to Configuration.py though. Casting to Decimal is fine. But, I'm not sure where you mean to use get(). I'm using it for getting the minloansize already I thought?

Comment (Evanito) [inline modules/Configuration.py]: https://github.com/Mikadily/poloniexlendingbot/blob/lending-rewrite/modules/Configuration.py#L40

This allows us to enforce minimums and the like.

File: docs/configuration.rst [hunk 5]
```diff
@@ -241,14 +241,14 @@ This can be configured in one of two ways.
 **Separate coin sections**
 
 This is an alternative layout for the coin config mentioned above. It provides the ability to change the minloansize per coin, but is otherwise identical in functionality.
-To use this configuration, make sure to comment out the line where coincfg is defined, then add a secion for each coin you wish to configure. These sections should come 
-at the end of the file, after the other options for the bot.
+To use this configuration, make sure to comment out the line where coincfg is defined, then add a section for each coin you wish to configure.
+.. warning:: These sections should come at the end of the file, after the other options for the bot.
```
Comment (Evanito) [inline docs/configuration.rst]: This builds as:

>To use this configuration, make sure to comment out the line where coincfg is defined, then add a section for each coin you wish to configure. .. warning:: These sections should come at the end of the file, after the other options for the bot. Configuration should look like this:

You need to skip a line between them so it looks like:
```
To use this configuration, make sure to comment out the line where coincfg is defined, then add a section for each coin you wish to configure.

.. warning:: These sections should come at the end of the file, after the other options for the bot.

Configuration should look like this:
```

Comment (laxdog) [inline docs/configuration.rst]: OK. Is there a way you're testing that when writing? I was just using vim, bit I'd like to have actually rendered what it'll look like in readthedocs. 

I suspect I'll run into this type of issue again.

Comment (Evanito) [inline docs/configuration.rst]: If you install sphinx through pip, you can do `build html` in the `/docs/` directory to build the documentation for you, just make sure not to commit it.

**Pull request merged**